week-1/implement/sum.js

```diff
@@ -0,0 +1,20 @@
+
+function sum(input) {
+  let output = [];
+  for (let i = 0; i < input.length; i++) {
+    if (typeof input[i] !== "string") {
+      output.push(input[i]);
+    }
+  }
+  let total = 0;
+  for (let i = 0; i < output.length; i++){
+    total = total + output[i];
+  }
+  return total;
+}
+
```

Comment:
In this exercise, I can see what you are trying to do here. But you don't really need to use two for loops to get this done, instead, you do the sum in line 6, so basically you should say if input[i] !== "string" then total = toal+input[i]

Let message me if you did not get it!